user-manual/metadata/premis.rst

```diff
@@ -4,11 +4,233 @@
 PREMIS metadata in Archivematica
 ================================
 
-Archivematica supports the entry of `PREMIS <http://www.loc.gov/standards/premis/>`_
-rights metadata during :ref:`transfer <transfer>` or :ref:`ingest <ingest>`.
-Rights entered via the GUI interface apply to the entirety of the SIP or transfer.
-Rights can also be entered at the object level by describing them in a rights.csv
-file and using the :ref:`Import metadata <import-metadata>` feature.
+Archivematica supports `PREMIS`_ metadata and currently implements `version 3`_.
+It captures extensive technical metadata about Objects and records actions
+(known as Events) taken on Objects by Agents. It also supports the addition of
+PREMIS Rights metadata. This page discusses how PREMIS is implemented in the AIP
+METS file.
+
+*On this page:*
+
+* :ref:`PREMIS Events metadata <premis-events>`
+* :ref:`PREMIS Events list <event-list>`
+* :ref:`PREMIS Agent metadata <premis-agent>`
+* :ref:`PREMIS Rights metadata <premis-rights>`
+
+.. _premis-events:
+
+PREMIS Events metadata
+======================
```

Comment:
These first-level heading adornments should be dashes `----` (it's inconsistent throughout the docs but I've been trying to standardize it over time - see [here](https://github.com/artefactual/archivematica-docs/blob/1.11/style_guide.md#section-headings) in the style guide)

Reply:
Good catch! My confusion makes sense if it was inconsistent, I copied what was already there. I've fixed the rest on the page.